node_modules/oae-mixpanel/lib/api.js

```diff
@@ -16,8 +16,11 @@
 var _ = require('underscore');
 var Mixpanel = require('mixpanel');
 
+var ContentAPI = require('./api.content');
+var UsersAPI = require('./api.users');
```

Comment:
Let's call these `MixPanelContentAPI` and `MixPanelUsersAPI`, just so they aren't confused with wide-spread familiarity of these being the actual `ContentAPI` and `UserAPI`